app/components/CategoriesNav/menu.module.css

```diff
@@ -15,6 +15,7 @@
   align-items: flex-start;
   margin: 5px;
   position: relative;
+  padding: 0px 10px 0px 24px;
```

Comment:
We're trying to use variables when we define numbers. Partially for conversion to mobile, partially so we can avoid numbers that aren't in the  design system like 10px. If you want you can change these but I'll still approve :).

Reply:
Ok, thanks. I'll adjust this PR to have variables and numbers that are in the design system.
If you want to merge to get the change on `stampy-redesign` though, go ahead and I can make the change in another PR.